internal/services/perf_test.go

```diff
@@ -0,0 +1,74 @@
+//go:build ci
+// +build ci
+
+package services_test
+
+import (
+	"context"
+	"sync"
+	"testing"
+	"time"
+
+	v1 "github.com/authzed/authzed-go/proto/authzed/api/v1"
+	"github.com/jzelinskie/stringz"
+	"github.com/rs/zerolog"
+	"github.com/stretchr/testify/require"
+
+	"github.com/authzed/spicedb/internal/datastore"
+	tf "github.com/authzed/spicedb/internal/testfixtures"
+	"github.com/authzed/spicedb/internal/testserver"
+	testdatastore "github.com/authzed/spicedb/internal/testserver/datastore"
+	"github.com/authzed/spicedb/internal/testserver/datastore/config"
+	dsconfig "github.com/authzed/spicedb/pkg/cmd/datastore"
+	"github.com/authzed/spicedb/pkg/tuple"
+	"github.com/authzed/spicedb/pkg/zedtoken"
+)
+
+func TestBurst(t *testing.T) {
+	blacklist := []string{
+		"spanner", // spanner emulator doesn't support parallel transactions
```

Comment:
Can we have the ID be pulled from the spanner package or would that create a weird dependency?